scripts/init.sh

```diff
@@ -5,8 +5,7 @@ set -e
 echo "*** Initializing WASM build environment"
 
 if [ -z $CI_PROJECT_NAME ] ; then
-   rustup update nightly
    rustup update stable
 fi
 
-rustup target add wasm32-unknown-unknown --toolchain nightly
\ No newline at end of file
+rustup target add wasm32-unknown-unknown --toolchain stable
```

Comment:
Shall we update the rust-toolchain file ?

Reply:
@simonsso what do you suggest?

Reply:
Rust toolchain file is updated but it could not modify the apt-get installed version when running benchmarks so this is only a reflection we nolonger need nightly

Reply:
I left this in because I was not sure if it was still in use in some corner of CI or something.